gobblin-compaction/src/main/java/gobblin/compaction/mapreduce/MRCompactionTask.java

```diff
@@ -34,6 +37,8 @@ public MRCompactionTask(TaskContext taskContext) throws IOException {
     this.suite = CompactionSuiteUtils.getCompactionSuiteFactory (taskContext.getTaskState()).
             createSuite(taskContext.getTaskState());
     this.dataset = this.suite.load(taskContext.getTaskState());
+    this.eventSubmitter = new EventSubmitter.Builder(this.metricContext, "gobblin.MRCompactionTask")
```

Comment:
"gobblin.MRCompactionTask" is the namespace. Please notify the wimd team that this is the new namespace they should query.